backend/ethereum/channel/depositor.go

```diff
@@ -27,6 +27,9 @@ type (
 	// Depositor is used by the `Funder` to deposit funds for ledger channel
 	// funding. Depositor are reusable such that one Depositor per asset is enough.
 	Depositor interface {
+		// Account returns the depositor account.
+		Account() accounts.Account
```

Comment:
If you want to store the account in the `Depositor`, you could also remove it from the `DepositReq` and remove this function.